org.jdownloader.JDownloader.desktop

```
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Type=Application
 Name=JDownloader
+Comment=JDownloader is a free, open-source download management tool
```

JDownloader is not entirely open-source unfortunately.
```suggestion
Comment=JDownloader is a free download management tool
```